.github/workflows/dev.yml

```diff
@@ -12,6 +12,9 @@ on:
 env:
   NODE_OPTIONS: "--max_old_space_size=4096"
 
+env:
```

Comment:
this appears to be duplicate of line 12 and 13

Reply:
+1